spec/spec_helper.rb

```diff
@@ -10,6 +10,24 @@
   end
 end
 
+def get_subcards_of_metric_value metric, company, content, year, source
```

Comment:
Looks like this method is more about 'creating/initializing' than 'getting'.
Anyway, I think you don't need that. There is an API for creating metrics with values and in the branch I'm working on it's now very powerful and much better documented than it used to be: https://github.com/xithan/wikirate/blob/all_metrics/mod/01_rating/lib/card/metric.rb#L34